ios/FluentUI/Vnext/ActivityIndicator/ActivityIndicatorModifiers.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+public extension ActivityIndicator {
+
+    /// Sets the accessibility label for the Activity Indicator.
+    /// - Parameter accessibilityLabel: Accessibility label string.
+    /// - Returns: The modified Activity Indicator with the property set.
+    func accessibilityLabel(_ accessibilityLabel: String?) -> ActivityIndicator {
+        state.accessibilityLabel = accessibilityLabel
+        return self
+    }
+
+    /// Sets the color of the Activity Indicator.
+    /// - Parameter color: UIColor instance to be set.
+    /// - Returns: The modified Activity Indicator with the property set.
+    func color(_ color: UIColor?) -> ActivityIndicator {
```

Comment:
I think this is a change that is maybe worth its own PR.
The foregroundColor takes a Color instead of a UIColor, which will require more changes in the state object.
On top of that there's the experimentation you just mentioned on the overriding portion.